pkg/terraform/util.go

```diff
@@ -34,6 +34,13 @@ func Configure(assetDir, renderedBackend string) error {
 		return errors.Wrapf(err, "Failed to create required terraform directory")
 	}
 
+	// TODO: Don't deploy self-hosted kubelet when `disable_self_hosted_kubelet`
```

Comment:
We should document, that changing `disable_self_hosted_kubelet` after cluster is created has no effect.

Reply:
Makes sense to me. Feel free to open an issue.